SeleniumProject/OpemText-samples/src/main/java/opentext/mbt/utils/Elements.java

```diff
@@ -0,0 +1,35 @@
+package opentext.mbt.utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class Elements {
+
+    public static RemoteWebDriver driver;
```

Comment:
I am pretty sure the driver won't be injected here, unless you add an empty test to this class.

Reply:
added changes